src/csr.cpp

```diff
@@ -92,6 +111,19 @@ CertificateSigningRequest CertificateSigningRequest::fromPEMFile(const std::stri
     return CertificateSigningRequest{_PEM_read_bio_X509_REQ(bio.internal())};
 }
 
+CertificateSigningRequest CertificateSigningRequest::fromDER(const std::vector<uint8_t> &derData)
+{
+    BioObject bio{BioObject::Types::MEM};
+    bio.write(derData);
+    return CertificateSigningRequest{_d2i_X509_REQ_bio(bio.internal())};
+}
+
+CertificateSigningRequest CertificateSigningRequest::fromDERFile(const std::string &filename)
```

Comment:
Why no "toDERFile" method?

Reply:
No particular reason. This doesn't exist for the other objects as well. Do you need this? Then I can add it.

Reply:
Not needed, I was wondering if it was forgotten as you added "fromDERFile"